bindings/go/README.md

```diff
@@ -17,12 +17,38 @@ provided there is enough buffer size to accomodate several of such blocks in one
 Read()/Write() request. (For example, see the source code of `mcp` utility and its `-bsz`
 option.)
 
-To use `go/mio` in your app 1) install motr-devel pkg and 2):
+## Installation
+
+Use go get, make sure you using Go version with [modules](https://github.com/golang/go/wiki/Modules) support.
+
+```Shell
+go get github.com/Seagate/cortx-motr/bindings/go
+```
+
+Next install `motr-devel` pkg and then import into sources.
 
 ```Go
 import github.com/Seagate/cortx-motr/bindings/go/mio
 ```
 
+Then init mio:
+
+```Go
+ha_addr := "192.168.63.52@tcp:12345:34:1"
+local_addr := "192.168.63.52@tcp:12345:33:1000"
+profile := "0x7000000000000001:0"
+process_fid := "0x7200000000000001:64"
+mio.Init(local_addr, ha_addr, profile, process_fid)
```

Comment:
You should mention that these parameters are optional. If they are not specified at mio.Init(), user should provide them via command-line parameters in his application, like `mcp` utility does.

Otherwise, it makes an impression that this is the only way to provide the parameters. (And ugly way, because it's hardcoded.)

Reply:
I will add another way which could pass args when running the app which import mio, this code just for illustrate usage, I would say typical case these values probably will be read from some properties file.